### PR TITLE
Fix breadcrumbs

### PR DESCRIPTION
**Pull request by janfaracik**
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by timja** [comment]
Quite a few still failed

File: src/main/resources/ath-container/run.sh [hunk 1]
```diff
@@ -45,7 +45,8 @@ if [ ! -f $war ]; then
     case "$war" in
         "latest")
             war=jenkins-latest.war
-            url=$mirrors/war/latest/jenkins.war
+	    # https://github.com/jenkinsci/jenkins/pull/6084
+            url=https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.327-rc31910.c126f7aa156c/jenkins-war-2.327-rc31910.c126f7aa156c.war
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:49]:
```suggestion
            url=https://repo.jenkins-ci.org/snapshots/org/jenkins-ci/main/jenkins-war/2.328-SNAPSHOT/jenkins-war-2.328-20220102.194634-1.war
```

File: src/main/resources/ath-container/run.sh [hunk 2]
```diff
@@ -45,7 +45,8 @@ if [ ! -f $war ]; then
     case "$war" in
         "latest")
             war=jenkins-latest.war
-            url=$mirrors/war/latest/jenkins.war
+	    # https://github.com/jenkinsci/jenkins/pull/6084
+            url=https://repo.jenkins-ci.org/snapshots/org/jenkins-ci/main/jenkins-war/2.328-SNAPSHOT/jenkins-war-2.328-20220102.194634-1.war
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:49]:
```suggestion
            url=https://ci.jenkins.io/job/Core/job/jenkins/job/PR-6084/37/artifact/org/jenkins-ci/main/jenkins-war/2.331-rc32008.f2e78735693e/jenkins-war-2.331-rc32008.f2e78735693e.war
```

**Comment by janfaracik** [comment]
Should *hopefully* pass now 🙏

File: src/main/resources/ath-container/run.sh [hunk 3]
```diff
@@ -45,7 +45,8 @@ if [ ! -f $war ]; then
     case "$war" in
         "latest")
             war=jenkins-latest.war
-            url=$mirrors/war/latest/jenkins.war
+	    # https://github.com/jenkinsci/jenkins/pull/6084
+            url=https://ci.jenkins.io/job/Core/job/jenkins/job/PR-6084/37/artifact/org/jenkins-ci/main/jenkins-war/2.331-rc32008.f2e78735693e/jenkins-war-2.331-rc32008.f2e78735693e.war
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:49]:
```suggestion
            url=https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.332-rc32038.7ef9976b_7d6d/
```

File: src/main/resources/ath-container/run.sh [hunk 4]
```diff
@@ -45,7 +45,8 @@ if [ ! -f $war ]; then
     case "$war" in
         "latest")
             war=jenkins-latest.war
-            url=$mirrors/war/latest/jenkins.war
+	    # https://github.com/jenkinsci/jenkins/pull/6084
+            url=https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.332-rc32038.7ef9976b_7d6d/
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:49]:
```suggestion
            url=https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.332-rc32038.7ef9976b_7d6d/jenkins-war-2.332-rc32038.7ef9976b_7d6d.war
```

**Comment by timja** [comment]
Few tests failing:

```
java-11-split3 / uninstall_plugin – core.PluginManagerTest
37s
java-11-split6 / build_multimodule – plugins.MavenPluginTest
55s
java-11-split5 / tie_job_to_specified_label – core.SlaveTest
```

The Batch ones are failing on master which I'll check before merging this

**Comment by timja** [comment]
Not able to reproduce `uninstall_plugin – core.PluginManagerTest`

Other failure was due to selenium clicking the far right hand side and bringing up the context menu

**Comment by timja** [comment]
Uninstall test is unrelated to this PR, also failed in https://github.com/jenkinsci/acceptance-test-harness/runs/4958889552

**Comment by timja** [comment]
Will merge https://github.com/jenkinsci/acceptance-test-harness/pull/740 if it passes

**Pull request closed without merging**